wpilibNewCommands/src/main/native/cpp/frc2/command/CommandPtr.cpp

```diff
@@ -192,6 +192,11 @@ CommandPtr CommandPtr::HandleInterrupt(std::function<void(void)> handler) && {
       });
 }
 
+CommandPtr CommandPtr::WithName(std::string name) {
```

Comment:
If you’re moving this, the function should be `&&` like other decorators to keep it from being used in lvalue context.